### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Vivdyuk**
- [DEMO LINK](https://vivdyuk.github.io/layout_snake/)
- [TEST REPORT LINK](https://vivdyuk.github.io/layout_snake/report/html_report/)

### CODE REVIEW

**Review by ArtemTopolskyi — CHANGES_REQUESTED** [review]
На широких экранах не соответствует условию:
image

image

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+  margin: initial;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+  align-content: stretch;
+}
+
+[class*="block"] {
```
Comment (ArtemTopolskyi) [inline src/style.css:19]: Просто задай class="block"

**Review by ArtemTopolskyi — CHANGES_REQUESTED** [review]

**Review by aionova — CHANGES_REQUESTED** [review]
On wide screens there still need to be not more than 3 blocks in a row. 
Screenshot 2021-05-31 at 15 09 59

**Comment by Vivdyuk** [comment]
> On wide screens there still need to be not more than 3 blocks in a row.
> Screenshot 2021-05-31 at 15 09 59

**Review by aionova — APPROVED** [review]
Good job!

**Comment by Vivdyuk** [comment]
> Good job!

Thanks!

**Pull request closed without merging**